frontend/src/utils/CustomerUtils.ts

```diff
@@ -10,7 +14,7 @@ const difference = (first: number, second: number) =>
   180 - Math.abs(Math.abs(first - second) - 180);
 
 export const randomColor = (
-  customers: Customer[] | undefined,
+  customers: Customer[] | RoadmapCreationCustomer[] | undefined,
```

Comment:
This could just take a list of colours, or `{ color: string }[]` which wouldn't require any changes in the callers